contracts/staking/ServiceStakingBase.sol

```diff
@@ -0,0 +1,278 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.21;
+
+// Multisig interface
+interface IMultisig {
+    /// @dev Gets the multisig nonce.
+    /// @return Multisig nonce.
+    function nonce() external returns (uint256);
+}
+
+// Service Registry interface
+interface IService {
```

Comment:
Should have their own files imo?

Reply:
Will be used when possible, as we don't want the bytecode of our original contracts changed if original interfaces are updated. Now I remember that was the reason not to add to existing files.

Reply:
True